.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: osx
   pool:
-    vmImage: macOS-10.15
+    vmImage: macOS-11
```

Comment:
Moving to macOS 11 may cause issues for LLVM 11's various library tests, which is what seems to be failing